logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/call/usecase/HasConferenceCallingActivatedUseCaseTest.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature.call.usecase
+
+import app.cash.turbine.test
+import com.wire.kalium.logic.configuration.UserConfigRepository
+import com.wire.kalium.logic.functional.Either
+import io.mockative.Mock
+import io.mockative.every
+import io.mockative.mock
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class HasConferenceCallingActivatedUseCaseTest {
+
+    @Test
+    fun givenOnlyDefaultConferenceCallingValue_thenDoNotReturnAnything() = runTest {
```

Comment:
Let's stick to `given..when..then` pattern for consistency 
https://wearezeta.atlassian.net/wiki/spaces/AR/pages/122520039/Code+Style+Guideline#Test-methods-naming
maybe :
givenOnlyDefaultConferenceCallingValue_whenUseCaseIsCalled_thenDoNotReturnAnything

Reply:
I dont know why I didnt put the `when` here, all other cases have it, fixed 👍